internal/federationdomain/endpoints/auth/auth_handler.go

```diff
@@ -210,7 +212,11 @@ func (h *authorizeHandler) authorize(
 	if requestedBrowserlessFlow {
 		err = h.authorizeWithoutBrowser(r, w, oauthHelper, authorizeRequester, idp)
 	} else {
-		err = h.authorizeWithBrowser(r, w, oauthHelper, authorizeRequester, idp)
+		var authorizeID string
+		authorizeID, err = h.authorizeWithBrowser(r, w, oauthHelper, authorizeRequester, idp)
+
+		h.auditLogger.Audit(plog.AuditEventUpstreamAuthorizeRedirect, r.Context(), nil,
```

Comment:
This doesn't (yet) check for `err != nil` since that's not test-driven

Reply:
It looks like the nil guard was since added. Resolving this conversation.